crates/typst/src/foundations/decimal.rs

```diff
@@ -29,7 +29,7 @@ use crate::World;
 /// raised).
 ///
 /// You can also convert any [integer]($int) to a decimal with the
-/// `{decimal(int)}` constructor, e.g. `{decimal(59)}`. However, note that
+/// `{decimal(int)}` constructor, e.g. `{decimal(69)}`. However, note that
```

Comment:
Why?

Reply:
We all know why. :)

<details><summary>In case you really don't know</summary>

There is a **very** popular meme about the number 69. Every time you hear it, you would say "nice". And since it already had the 9 in ones and off-by-one in tens, I think it's a great opportunity to insert a drop of modern humor in the docs (because it also doesn't harm anyone or anything like that, it's a purely positive thing). I think a lot of people, who know about this thing/meme/trend, would really appreciate Typst having it in the docs. You can say it can be one of hidden gem. ;)

</details>

Reply:
Using 69 in an example is just unprofessional

Reply:
People that don't know what this all about would disagree with you (at least IMO). It's just a regular integer like any other (like 59, but bigger). Nothing special. But if someone _knows_ and sees it in the docs, I'm sure it will put a little smile on their faces. It certainly does put a smile on _mine_ every time _I_ see it anywhere.

Reply:
I'd prefer for this to be left as-is.

Reply:
> I'd prefer for this to be left as-is.

I'm not 100% sure what this refers to. To my change or to the initial docs? Do I revert the change?